cpp/cmake/Modules/JitifyPreprocessKernels.cmake

```diff
@@ -16,8 +16,6 @@
 
 cmake_minimum_required(VERSION 3.18)
 
-file(MAKE_DIRECTORY "${CUDF_GENERATED_INCLUDE_DIR}/include/jit_preprocessed_files")
-
 # Create `jitify_preprocess` executable
 project(jitify_preprocess VERSION 2.0 LANGUAGES CXX CUDA)
```

Comment:
Need to remove this line too.

Reply:
we're `include`ing this file in cudf's CMakeLists, so this actually ends up setting cudf's version, which we don't want. :disappointed: